docs/src/pages/components/data-grid/filtering/filtering.md

```diff
@@ -5,32 +5,36 @@ components: DataGrid, XGrid
 
 # Data Grid - Filtering
 
-<p class="description">Filtering helps view particular or related records in the data grid.</p>
+<p class="description">Filtering helps to view a subset of the records based on a criteria.</p>
 
-## Column filters
+## Basic filter
 
-### Basic filter
+Column filters can be set using the column menu and clicking the Filter menu item.
+Alternatively, if the grid has the toolbar displayed, you just need to click on the Filters button.
 
-Column filters can be set using the column menu and clicking the filter menu item.
-Alternatively, if the grid has the toolbar displayed, just need to click on the filter button.
+The filter applied to a column can be pre-configured using the `filterModel` prop:
 
-A filtered column can be can pre-configured using the `filterModel` prop:
-
-{{"demo": "pages/components/data-grid/filtering/BasicFilteringGrid.js", "bg": "inline"}}
+```jsx
+<DataGrid
+  filterModel={{
+    items: [{ columnField: 'commodity', operatorValue: 'contains', value: 'rice' }],
+  }}
+/>
+```
 
-### Toolbar
+{{"demo": "pages/components/data-grid/filtering/BasicFilteringGrid.js", "bg": "inline", "defaultCodeOpen": false}}
 
-In addition to the column menu that allows users to apply a filter, you can also show a toolbar:
+## Native filters
```

Comment:
We use this terminology in https://deploy-preview-1671--material-ui-x.netlify.app/components/data-grid/columns/#column-types. I personally find it less confusing, I wonder what the others think.

```suggestion
## Predefined filters
```

Reply:
I changed to "predefined". The page you linked uses both terms. We should keep just one. I'll fix it in another PR to improve the Columns page.

<img width="500" src="https://user-images.githubusercontent.com/42154031/118494055-1f683600-b6f8-11eb-9667-fb63f15c2463.png" alt="image" style="max-width:100%;">

Reply:
The usage of "native" here might be used as a synonym, to avoid the repetition.